src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Get started with Azion API
+description: Learn more about oficial endpoints, methods, and API use cases of the Azion RESTful API.
+meta_tags: azion, rest api, restful api, azion restful api
+namespace: documentation_products_azion_api_overview
+permalink: /documentation/products/azion-api-overview/
+---
+
+This documentation will guide you on the first steps of using the **Azion API**.
+
+## API overview
+
+The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).
```

Comment:
```suggestion
The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. The complete Azion API collection is on Postman

<Button href="https://api.azion.com/" text="go to Azion API collection"></Button>

Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).
```

Reply:
there's a link to the intro right after, but it seemed importante to clarify it's on Postman and add the button. we're using it in most docs!